Makefile.dep

```diff
@@ -48,6 +48,10 @@ ifneq (,$(filter sixlowpan,$(USEMODULE)))
   USEMODULE += vtimer
 endif
 
+ifneq (,$(filter ng_ipv6_hdr,$(USEMODULE)))
+  USEMODULE += ng_inet_csum
```

Comment:
Totally unrelated to this PR: The prefix NG sucks. Whats next? NNG -- "the next new negeration"? After that ENNG "even newer next generation"? UMNGT -- "ultimate mighty next generation thingy"?